BetterMD/elements/document.py

```
@@ -2,6 +2,7 @@
 
 if t.TYPE_CHECKING:
     from .symbol import Symbol
+    from ..typing import ATTR_TYPES
```

Ruff (0.8.2)

`..typing.ATTR_TYPES` imported but unused

Remove unused import: `..typing.ATTR_TYPES`

(F401)